src/ps33.js

```diff
@@ -0,0 +1,91 @@
+const auth = React.useMemo(
+    () => ({
+      login: async (email, password) => {
+        const { data } = await axios.post(`${BASE_URL}/auth/local`, {
+          identifier: email,
+          password,
+        });
+        const user = {
+          email: data.user.email,
+          token: data.jwt,
+          id: data.user.id,
+        };
+        dispatch(createAction('SET_USER', user));
+      },
+      logout: async () => {
+        dispatch(createAction('REMOVE_USER'));
+      },
+      register: async (email, password) => {
+        // await Sleep(2000);
+        const response = await axios.post(`${BASE_URL}/auth/local/register`, {
+          username: email,
+          email,
+          password,
+        });
+
+        const { data } = response;
+        const user = {
+          email: data.user.email,
+          token: data.jwt,
+        };
+        dispatch(createAction('SET_USER', user));
+      },
+      addToCart: (oneBook) => {
+        let contains = false;
+        let count = 1;
+        totalPrice += oneBook.Price;
+        const book = Object.assign(oneBook);
+        cart.forEach((bookO) => {
+          if (bookO.id === oneBook.id) {
+            contains = true;
+            // eslint-disable-next-line no-plusplus
```

Comment:
Do not disable eslint rules, this could cause errors in the future if another dev will work on this code